### PR TITLE
nebula + gradle version bump

### PR DESCRIPTION
**Pull request by syeboah5**
### Context

nebula and gradle version bumps, old versions not allowing compilation

### CODE REVIEW

**Review by sundargates — APPROVED** [review]

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -15,6 +15,6 @@
 #
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.8.3-bin.zip
```
Comment (Andyz26) [inline gradle/wrapper/gradle-wrapper.properties:18]: do you need to update the jar file in this folder too?

Comment (Andyz26) [reply gradle/wrapper/gradle-wrapper.properties:18]: nvm looks like oss version can handle this correctly.

File: .github/workflows/nebula-publish.yml [hunk 2]
```diff
@@ -13,6 +13,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v1
+      - run: |
```
Comment (Andyz26) [inline .github/workflows/nebula-publish.yml:16]: is this the recommended setup?

Comment (syeboah5) [reply .github/workflows/nebula-publish.yml:16]: yeah recommended from JVM team

File: .github/workflows/nebula-publish.yml [hunk 3]
```diff
@@ -13,6 +13,9 @@ jobs:
...
+            git config --global user.name "Netflix OSS Maintainers"
+            git config --global user.email "netflixoss@netflix.com"
```
Comment (hmitnflx) [inline .github/workflows/nebula-publish.yml:18]: nit: could you pl update the details as follows?
1. user.name - Mantis OSS Maintainers
2. user.email — mantis-oss-dev@googlegroups.com

Comment (syeboah5) [reply .github/workflows/nebula-publish.yml:18]: will do

**Review by hmitnflx — APPROVED** [review]

**Pull request merged**